ote/src/clj/ote/services/pre_notices/attachments.clj

```diff
@@ -49,6 +49,10 @@
               (fn [out]
                 (io/copy (:input-stream s3-file) out)))})))
 
+(defn delete-from-aws [db {bucket :bucket :as config} attachments]
+  (doseq [{id ::transit/id file-name ::transit/attachment-file-name}  attachments]
+    (s3/delete-object bucket (str id "_" file-name))))
```

Comment:
Better to create a function for generating bucket file keys. This is already used in "upload attachment" "download attachment" and "delete-from-aws" If the filekey format changes at some point, we'd have to remember to change it for all those places.

Reply:
Fixed. Key-file function created for common use.